src/renderer/controllers/playback-controller.js

```diff
@@ -34,12 +34,20 @@ module.exports = class PlaybackController {
         else this.play()
       })
     } else {
+      let initialized = false
```

Comment:
I had an idea to set `torrentSummary.mostRecentFileIndex` instead of using the `initialized` variable and remove the index param from `openPlayer`, but I don't think it's the best solution after all.